src/discord.coffee

```diff
@@ -55,6 +55,8 @@ class DiscordBot extends Adapter
         @robot.client = @client
         @client.on 'ready', @.ready
         @client.on 'message', @.message
+        @client.on 'guildMemberAdd', @.enter
+        @client.on 'guildMemberRemove', @.leave
```

Comment:
We should likewise handle `guildBanAdd` since it will often represent a user leaving.